pkg/workload/ycsb/zipfgenerator_test.go

```diff
@@ -30,7 +29,7 @@ var gens = []params{
 func TestCreateZipfGenerator(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	for _, gen := range gens {
-		rng := rand.New(rand.NewSource(uint64(timeutil.Now().UnixNano())))
+		rng := rand.New(rand.NewPCG(rand.Uint64(), rand.Uint64()))
```

Comment:
Interesting. The global source is seeded with AT_RANDOM [1].

[1] https://github.com/golang/go/blob/391dde29a37f3fd450f7d61e3f220930e0164b89/src/runtime/os_linux.go#L300-L314